augly/image/__init__.py

```diff
@@ -212,5 +215,6 @@
     "scale_intensity",
     "sharpen_intensity",
     "shuffle_pixels_intensity",
+    "Skew",
```

Comment:
Again, you need to add "skew" & "skew_intensity" to this __all__ list in this file.

Reply:
Please check I've just updated.